security-force-monitor/sfm-cms
# Refactor downloads to support source/confidence values

## Overview

Refactor the data download views to support optional source/confidence values in downloads, and expose new fields and fieldnames in downloads.

Connects #660, #661.

## Testing Instructions

- Update your app container image to install new requirements: `docker-compose build app`
- Pull down this branch and create the export tables:

```
docker-compose run --rm app ./manage.py make_materialized_views
```

- Start the app server with `docker-compose up`
- Navigate to http://localhost:8000/en/download/
    - For each download type, test the download with and without sources/confidence values, and confirm that you get the right set of fields in each case
